ci/Jenkinsfile.nightly.integration

```diff
@@ -82,3 +90,16 @@ pipeline {
   }
 }
 
+def runTestCases(test_cases, iterations) {
```

Comment:
looks like this function name should be `runTests`?

Reply:
Thanks for catching that, I didn't test this in ci yet. Will do a test run once libp2p PRs are merged to master.